DashWallet/Sources/UI/Explore Dash/Views/DWExploreTestnetContentsView.h

```diff
@@ -31,4 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nullable, nonatomic, copy) NSString *subtitle;
 @end
 
+@interface DWExploreCrowdNodeContentsViewCell : DWExploreTestnetContentsViewCell
```

Comment:
I figured it might be better to extend `DWExploreTestnetContentsViewCell` specifically for CrowdNode item instead of adding the APY stub to it directly.